files/image_config/bash/bash.bashrc

```diff
@@ -56,3 +56,12 @@ fi
 
 # Automatically log out console ttyS* sessions after 15 minutes of inactivity
 tty | grep ttyS >/dev/null && TMOUT=900
+
+# if SSH_TARGET_CONSOLE_LINE was set, attach to console line interactive cli directly
+if [ -n "$SSH_TARGET_CONSOLE_LINE" ]; then
+    # enter the interactive cli
+    sudo connect line $SSH_TARGET_CONSOLE_LINE
```

Comment:
what is the user does not have sudo privilege? what will happen here?

Reply:
A short error message will display:
```
<user_name> is not in the sudoers file.  This incident will be reported.
```
And then the session will close.

Do we need test if the user is in sudoer list before run this command?

Reply:
i think normal user should be able to login via the console line, should not need sudo user.

Reply:
No. The connect command require admin privilege because it need to talk with \dev\ttyUSB*

Reply:
Can we fix that somehow? Current console access doesn't require elevated privilege. I think.

Reply:
We need to add user to group `dialout` and full reboot.
`sudo adduser <user> dialout`

Or apply UDEV rule:
https://askubuntu.com/questions/112568/how-do-i-allow-a-non-default-user-to-use-serial-device-ttyusb0
And set the usb device file to 666

I prefer udev rule because we already have one for console ports and we can control which device can expose without admin privilege need